trade-finance-manager-ui/server/controllers/case/index.js

```diff
@@ -49,6 +52,7 @@ const getCaseDeal = async (req, res) => {
     amendments,
     amendmentsInProgress,
     hasAmendmentInProgress,
+    showDealCancelButton: dealCancellationEnabled(deal.dealSnapshot.submissionType, user),
```

Comment:
can submissionType be dereferenced from deal please

Reply:
I've pulled this out 👍